### PR TITLE
Add dynamic popup window

### PR DESCRIPTION
**Pull request by Wes-Isaac**
Add dynamic popup window when the button on the projects section is clicked.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,62 @@
+const project = {
+  title: 'Keeping track of hundreds of components',
+  image: 'images/snapShootPortfolio.png',
```
Comment (Alexoid1) [inline index.js]: - [ ] The image is not loading. Kindly fix the route for the image.
Captura de pantalla_2021-10-06_01-53-08

- [ ] The modal window has to be responsive for mobile devices.
Captura de pantalla_2021-10-06_01-54-05

File: index.js [hunk 2]
```diff
@@ -1,3 +1,62 @@
+const project = {
```
Comment (Alexoid1) [inline index.js]: - [ ] Use an array of objects to store the projects. This will make it easier to update your portfolio with new projects.

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
# Hi @Wes-Isaac,

Impressive project!
You are doing a great job so far, and the project is looking good.
- No linter errors.
- Descriptive README.MD file.
- Follow Git-hub flow.

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes :recycle:
 - For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Isaac, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer(s). There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**